engine/src/build_overrides/angle.gni

```diff
@@ -18,7 +18,7 @@ if (!is_fuchsia) {
 }
 
 # Overrides for ANGLE's dependencies.
-angle_abseil_cpp_dir = "//flutter/third_party/abseil-cpp"
+angle_abseil_cpp_dir = "//third_party/abseil-cpp"
```

Comment:
Remind me - is this the top of flutter/flutter or is it the engine/src/flutter folder?

If we start storing files used to build the engine in root third_party, we need to update the content hash script.